community/planning/oauth2_rest_api_authentication.md

```diff
@@ -179,6 +187,25 @@ When biome features are enabled, the user's token and other details will be
 added to the biome `OAuthUserStore`; see [Biome OAuth Integration]({% link
 community/planning/biome_oauth_user_store.md %}) for more details.
 
+#### Logout Route
+
+The logout route is used by the browser application to remove the user's stored
+tokens from underlying storage. After the request has been authenticated by the
+middleware component, the Splinter REST API attempts to perform the configured
+operation to remove the user's access and refresh tokens. If this operation is
+successful, the Splinter REST API will respond with `200 Ok`.
+
+OAuth user storage is represented by the `OAuthUserInfoStore` trait, which
+defines methods to save and remove user information. When biome features are
+enabled, the `OAuthUserInfoStore` is backed by the biome `OAuthUserStore`; see
+[Biome OAuth Integration]({% link community/planning/biome_oauth_user_store.md %})
+for more details. If underlying storage has not been configured, no-op versions
+of these methods are used instead.
+
+If underlying storage has not been configured, once the request has been
+authorized by the middleware component, no storage operation is performed and
+the Splinter REST API will respond with `200 Ok`.
```

Comment:
This paragraph can probably be removed; this info is already covered by the previous paragraphs.